source/transfer_list.rst

```diff
@@ -343,6 +343,23 @@ Inputs:
 
    #. Add `align8(te.hdr_size + te.data_size)` to `te_base_addr`.
 
+
+Adding a void TE
```

Comment:
I would call this "Overwriting part of the TL with a void TE", because this isn't really "adding" a TE (to the end) the way the normal "Adding a new TE" operation does.

I would maybe put this at the end or at least further down in this section, because it's not really a useful operation on its own, it's only a common step for two others, and it's weird for the reader to see this first.

Reply:
I kind of agree with this but would go one step further. If we're adding an "Overwriting a TE" section then I think this it's unnecessary to have this operation also embedded within the existing "Adding a TE" section. So I would remove that bit from "Adding a TE", rename it to "Append a TE", and inline this "Overwriting part of the TL with a void TE" operation within the "Overwriting a TE" section.

Reply:
On 2nd thoughts, you might still need this operation in 2 places ("Overwriting a TE" and "Removing a TE"), in which case it might better in its own section. But still it should be below the operations that use it.